modules/Story/Story.tsx

```diff
@@ -31,47 +30,25 @@ function Story({ story }: Props) {
         return null;
     }
 
-    const { title, subtitle, content, format_version, categories, links } = story;
-    const headerImage = story.header_image ? JSON.parse(story.header_image) : null;
-    const hasHeaderImage = Boolean(headerImage);
+    const { categories, links } = story;
     const hasCategories = categories.length > 0;
+    const nodes = JSON.parse(story.content);
 
     return (
         <Layout>
             <StorySeo story={story} noindex={!isAnalyticsEnabled} />
             <article className={styles.story}>
-                <div
-                    className={classNames(styles.container, {
-                        [styles.withImage]: hasHeaderImage,
```

Comment:
I believe this class can be dropped from the stylesheet, since it was there for when the image was above the title/subtitles

Reply:
Do you mean the `container` or `withImage` class? Because I've removed the `withImage`.